index.html

```diff
@@ -1 +1,146 @@
+                    <a class="mobile-menu-items sponsor-section" href="#sponsor">Sponsor</a>
+                    <a class="mobile-menu-items news-section" href="#">News</a> 
+                </div>
+            </nav>
 
+        </header>
+        <main>
+            <section class="homePage">
+                <div class="mainContent">
+                    <div class="headingContainer">
+                        <h2 class="greetingHeading">"Hello!Beauty  world"</h2>
+                        <h1 class="titleHeading">SUMMIT OF BEAUTY 2022</h1>
+                    </div>
+                    <div class="mainDescription">
+                        <p>Connect with the best experts, in real life, at our summit.
+                         Be part of this unique experience and listen to their
+                          life-changing advice in person!
+                        </p>
+                    </div>
+                    <div class="date-venue">
+                        <p class="date">2022.10.15(THU) ~ 16(FRI)</p>
+                        <p>@ National Museum of Dubai, Art Center Dubai and more</p>
+                    </div>
+                </div>
+            </section>
+            <section class="programSection" id="Program">
+                <div class="program">
+                    <h2>Main Program</h2>
+                    <div class="line"></div>
+                    <div class="programContainer">
+                        <div class="programItem">
+                            <img src="./Assets/makeup.svg" alt="lecture">
+                            <h3>Make-up</h3>
+                            <p>Handpicked from the most reputable brands to provide you the best</p>
+                        </div>
+                        <div class="programItem">
+                            <img src="./Assets/skincare.svg" alt="exhibition">
+                            <h3>Skin care</h3>
+                            <p>More than just skin deep, Vegan and cruelty-free</p>
+                        </div>
+                        <div class="programItem">
+                            <img src="./Assets/haircare.svg" alt="forum">
+                            <h3>Hair care</h3>
+                            <p>Say goodbye to bad chemicals and unhealthy ingredients.</p>
+                        </div>
+                        <div class="programItem">
+                            <img src="./Assets/workshop.svg" alt="workshop">
+                            <h3>Workshop</h3>
+                            <p>More panels and workshops than last year</p>
+                        </div>
+                        <div class="programItem">
+                            <img src="./Assets/edit-regular.svg" alt="ignite">
+                            <h3>Q & A</h3>
+                            <p>This is your chance to be part of the conversation.</p>
+                            
```

Comment:
Each of the program items are articles hence it will be better to put them in article tags for SEO optimization and accessibility factors.